test-client/index.ts

```diff
@@ -19,17 +18,27 @@ engine.registerOnGameFinished((engine) => console.log(engine.players));
 engine.startGame();
 
 for (let i = 0; i < 10; ++i) {
-  const position = positionDice.roll();
-  const size = sizeDice.roll();
+  if (engine.isFinished) {
+    break;
+  }
+
+  const position = positionDices.map((d) => d.roll());

```

Comment:
Actually, It is not a good idea to use SimpleDice class instances for generating position. Generate them directly from random.

Reply:
Actually, it is just a test project which checks how all features of game engines work together. It doesn't matter how random coordinates and sizes are generated indeed.

Reply:
I used dices just because they already wrapped generation of random integer numbers in some range that's all.